scholia/api.py

```diff
@@ -156,7 +156,7 @@ def wb_get_entities(qs):
         'User-Agent': 'Scholia',
     }
     response_data = requests.get(
-        'https://www.wikidata.org/w/api.php',
+        config.get('servers', 'webserviceURL'),
```

Comment:
This variable is missing from the defaults. Shouldn't it be "WebserviceURL" if the CamelCase is followed?

Reply:
This was one of the things to fix